packages/core/src/Cardano/types/StakePool/StakePool.ts

```diff
@@ -106,57 +95,55 @@ export class StakePoolEpochRewards {
 
   /**
    * The epoch number at which these rewards were calculated.
+   *
+   * @deprecated Use `epochNo` instead
    */
-  epoch: EpochNo;
+  epoch?: EpochNo;
```

Comment:
This is a breaking change, please reflect in the commit message

Reply:
9bb6579a4 should solve this

Reply:
The commit message is now a little confusing in terms of CHANGELOG, as it will say sth like:
Breaking changes:
- core: prepares the types for typeorm stake pool provider w…

A more accurate commit message would be something like that:
```
refactor(core)!: prepares the types for typeorm stake pool provider w…

BREAKING CHANGE: deprecated StakePoolEpochRewards.epoch
- deprecated StakePoolEpochRewards.memberROI
```

Reply:
d476e6e2d should solve this

Reply:
This formatting will result in a change log like:
```
Breaking changes:
- - Deprecated QueryStakePoolsArgs.apyEpochsBackLimit
- ...
```

Because the 1st breaking change must be inline :sweat_smile:

Reply:
f79314b98 should solve this